### ISSUE TITLE
Documentation: Primer uses deprecated 

### ISSUE THREAD

**Issue by daldred** [mavoweb/mavo#953]
In the documentation  at mavo.io, the Primer shows how collections work using the 'mv-multiple' attribute.  The Collections documentation states that mv-multiple is deprecated in favour of mv-list.  However the Collections documentation also uses 'mv-multiple' further down the page under the heading "Simple and complex collections", with no information on how this works with 'mv-list'.  

It would be good to update the Primer and the "Simple and complex collections" section of the Collections page to show the use of mv-list, and move the deprecated approach down to page.

**Comment by LeaVerou** [comment]
You are right that the documentation as it currently stands is pretty …confused.
The reason is that we have not released a stable version for a while, as the development version has introduced some regressions we have not been able to fix. `mv-list` is currently only present in the development version, so `mv-multiple` cannot really be properly "deprecated" until there is a stable version that includes `mv-list`. 

@DmitrySharabin it may be a good idea to introduce a label for the regressions that would be really essential to fix before we can release a v0.3.0, so we know where we stand. It's been way too long that we've been in this state… 😕

**Comment by DmitrySharabin** [comment]
> it may be a good idea to introduce a label for the regressions that would be really essential to fix before we can release a v0.3.0

What if we mark those issues (regressions) with the existing `priority: high` label? Or I would suggest a new `release: urgent` label. Or `release: essential`?

We also have a couple of PRs that might become a part of a new release. Should we label them too?

**Comment by LeaVerou** [comment]
> > it may be a good idea to introduce a label for the regressions that would be really essential to fix before we can release a v0.3.0
> 
> What if we mark those issues (regressions) with the existing `priority: high` label? Or I would suggest a new `release: urgent` label. Or `release: essential`?
> 
> We also have a couple of PRs that might become a part of a new release. Should we label them too?

I think it's fine to use a combination of priority: high and appropriate milestones (currently there are a bunch of stuff on the v0.3.0 milestone which are not essential for release)

**Comment by DmitrySharabin** [comment]
Ok. Let me add labels and milestones to the regressions I believe are essential, so we could iterate over them, and bump all other new features to v0.4.0.
